src/components/ProjectCard/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import BaseText from "../BaseText";
+import Label from "../Label";
+import "./ProjectCard.css";
+
+function ProjectCard() {
```

Comment:
@AbdurrahmanSogbesan this component has to take in props. Youre currently hard coding the title, description and labels. Thos should be passed in as props.

Reply:
I know. I just did it like that for now to see the result